### PR TITLE
About: Integrate "Vision and Goals" from wiki, update team

### PR DESCRIPTION
**Pull request by jzaefferer**
This integrates the content from http://wiki.jqueryui.com/w/page/12138130/Vision%20and%20Goals

Under Collaborative design process, I've dropped the sentence referencing the wiki and forum. Under Internationalization and localization, I've dropped the last sentence as well (mentioned datepicker).

I've updated various links to point to the same site instead of the wiki, both in the new paragraphs and in the existing team descriptions. I've talked to Kris and moved him to the Contributors list. I've talked to Rafael and updated his description accordingly.

I've used the `build-posts` task to check the "final" HTML in the browser, but since I can't deploy the site locally, I don't know what it looks like with the theme.

### CODE REVIEW

File: page/about.html [hunk 1]
```diff
@@ -4,21 +4,48 @@
...
+
+<h3>Progressive enhancement</h3>
+
+Since we're developing non-native HTML controls, widgets should be built in such a way that users on browsers and devices that are unable to support Javascript can still interact with the website or application. Most of the current set of UI widgets follow the best practice of progressive enhancement and we're working to extend that pattern to widgets that don't. In most cases, advanced components can be generated from HTML primitives (i.e. slider from select menu, radio set, or text input), and as the UI library paired with the most popular JavaScript library in the world, we should lead by example in this area. Any UI widget that sits within the flow of a form should be able to store data using semantic HTML elements so the form can be submitted or serialized normally. jQuery UI components should follow the unobtrusive practices put forth by jQuery itself, and should attempt to be forward-looking in its attempts to normalize across browsers and devices (test for features/bugs, not browser sniffing).
```
Comment (scottgonzalez) [inline page/about.html]: Javascript -> JavaScript

File: page/about.html [hunk 2]
```diff
@@ -4,21 +4,48 @@
...
+
+<h3>Deep accessibility support</h3>
+
+Widgets should also be accessible to JavaScript-capable users who have disabilities such as blindness or deafness (should we ever venture into the arena of audio/video integration, for instance). We attempt to make components accessible through the use of semantic HTML elements within components and following the guidelines specified in the WAI-ARIA spec. Any image-based actions within widgets should provide text equivalents (close icons, expand/collapse icons, etc. should have title attributes at the least).
+
+<h3>Internationalization and localization</h3>
+
+Given the global audience for jQuery UI, we should embrace the ability to make our plugins work in a variety of languages and cultures.  By abstracting language away from the core plugin code and providing options for supporting cultural differences (right > left reading orientation, date and currency formats, iconography, etc.) we can build in the flexibility needed for a global community.
```
Comment (scottgonzalez) [inline page/about.html]: "right > left" -> "right to left"

File: page/about.html [hunk 3]
```diff
@@ -60,3 +87,4 @@
 <h3><a href="http://ralphwhitbeck.com/">Ralph Whitbeck</a></h3>
 <h3><a href="http://keith-wood.name/">Keith Wood</a></h3>
 <h3><a href="http://rdworth.org/">Richard D. Worth</a></h3>
+<h3><a href="http://krisborchers.com/">Kris Borchers</a></h3>
```
Comment (scottgonzalez) [inline page/about.html]: Missing newline.

**Comment by jzaefferer** [comment]
Updated. Will see if we need any other updates before merging it.

**Comment by jzaefferer** [comment]
And one more update, should be good to land now.

File: page/about.html [hunk 4]
```diff
@@ -60,3 +84,5 @@
...
+<h3><a href="http://gnarf.net/">Corey Frang</a></h3>
```
Comment (scottgonzalez) [inline page/about.html]: These should be alphabetical by last name.

**Comment by jzaefferer** [comment]
Updated, contributors are now sorted.

**Comment by arschmitz** [comment]
Should I perhaps be listed here as well?

**Comment by jzaefferer** [comment]
Yes, you should. Can you provide me with the blurb, in HTML?

**Comment by scottgonzalez** [comment]
I merged this because I didn't want to wait any longer to be able to update the wiki. We can add Alex at any point.

**Comment by arschmitz** [comment]
Yeah no need to wait on me

**Comment by jzaefferer** [comment]
Fun, I didn't get a merge/close notification email, because "I merged it myself", except I haven't, it was Scott.

**Pull request closed without merging**